docker-mixin/dashboards/docker/variables.libsonnet

```diff
@@ -0,0 +1,66 @@
+local grafana = import 'grafonnet/grafana.libsonnet';
+
+{
+    ds: {
+      docs: 'Prometheus datasources',
+        spec: {
+        current: {
+          text: 'default',
+          value: 'default',
+        },
+        hide: 0,
+        label: 'Data Source',
+        name: 'prometheus_datasource',
+        options: [],
+        query: 'prometheus',
+        refresh: 1,
+        regex: '',
+        type: 'datasource',
+      },
+    },
+
+    job: {
+      docs: 'Jobs available',
+      spec: grafana.template.new(
+        'job',
+        '$prometheus_datasource',
+        'label_values(machine_scrape_error, job)',
+        label='Job',
+        refresh='load',
+        multi=true,
+        includeAll=true,
+        allValues='.+',
+        sort=1,
+      ),
+    },
+
+    instance: {
+      docs: 'Available instances (nodes)',
+      spec: grafana.template.new(
+        'instance',
+        '$prometheus_datasource',
+        'label_values(machine_scrape_error{job=~"$job"}, instance)',
```

Comment:
Sometimes variables depend on config (instead of hardcoded job=~"$job" it could be dynamic label selector). Does this packaging provide a way to provide custom labelsSelector from config?

Reply:
This is entirely up to us to define, so I'm sure it can!